src/components/auth/signup/Signup.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import createUser from '../../../actions/signupActions/signup';
+import DisplayMessage from './DisplayMessage';
+import './Signup.css';
+
+class Signup extends Component {
+  state = {
+    username: '',
+    email: '',
+    password: '',
+    confirmPassword: '',
+    message: false,
+  }
+
+  onChange = event => this.setState({ [event.target.name]: event.target.value });
+
+  comparePassword = () => {
```

Comment:
Nice implementation @marcdomain . Love the fact that you are checking if the passwords match.